### PR TITLE
Server gets data from database

### PR DESCRIPTION
**Pull request by enkhbatMunkhbold**

### CODE REVIEW

File: client/src/components/app.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </li>
+            )
+        }
+    }
+}
+
+export default App;
```
Comment (jonathantch) [inline client/src/components/app.jsx]: Need to have a blank new line at the end according to ESlint.

**Comment by tinaLikesCheese** [comment]
Everything looks good, but you need to handle your response/request errors.

File: database/reviews.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const db = require('./index.js');
+// const db = require('./index.js');
```
Comment (jonathantch) [inline database/reviews.js]: Remove comment or label it if it's going to be used later

File: server/index.js [hunk 3]
```diff
@@ -1,39 +1,23 @@
...
-app.post('/', (req, res, next) => {
-    res.status(201).json({
-        message: 'POST reviews are created'
-    });
+app.get('/reviews', (req, res) => {
+    Review.find(function(err, Review) {
+        if(err) {
+            console.error(err);
+        }
+        // console.log(Review);
```
Comment (jonathantch) [inline server/index.js]: remove console.log

**Pull request merged**